indico/modules/events/logs/client/js/containers/LogEntryModal.js

```diff
@@ -26,6 +33,11 @@ const mapDispatchToProps = dispatch => ({
   nextEntry: () => {
     dispatch(viewNextEntry());
   },
+  relatedEntries: metadataQuery => {
+    dispatch(setMetadataQuery(metadataQuery));
+    dispatch(setPage(1));
+    dispatch(fetchLogEntries());
```

Comment:
if you close the modal, `await` this dispatch to only close the modal after loading finished